index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="pt-br">
```
O que você acha de colocar na nossa linguagem padrão ? :)